cypress/e2e/settings.cy.js

```diff
@@ -1,32 +1,92 @@
 /// <reference types='cypress' />
 /// <reference types='../support' />
+import SignInPageObject from '../support/pages/signIn.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
+import SettingsPageObject from '../support/pages/settings.pageObject';
+import faker from 'faker';
+import {
+  updateSuccessMessage,
+  confirmationMessage
+} from '../plugins/alertMessages';
+
+const signInPage = new SignInPageObject();
+const homePage = new HomePageObject();
+const settingsPage = new SettingsPageObject();
 
 describe('Settings page', () => {
-  before(() => {
+  let user;
 
+  before(() => {
+    cy.task('db:clear');
   });
 
   beforeEach(() => {
+    cy.task('generateUser').then((generateUser) => {
+      user = generateUser;
+
+      signInPage.visit();
+
+      cy.register(user.email, user.username, user.password);
```

Comment:
Registration function should be separated out of beforeEach hook and put into before hook because we don't need to register a new user before each test, we just need to log in each time.

Reply:
done